src/main/java/appeng/tile/AEBaseTile.java

```diff
@@ -434,7 +434,16 @@ public void disableDrops() {
     }
 
     public void saveChanges() {
-        if (this.world != null) {
+        if (this.world == null) {
+            return;
+        }
+
+        // Clientside is marked immediately as dirty as there is no queue processing
+        // Serverside is only queued once per tick to avoid costly operations
+        // TODO: Evaluate if this is still necessary
+        if (this.world.isRemote) {
+            this.markDirty();
```

Comment:
Not necessary to mark chunks dirty on the client-side, this probably won't be a method that gets called from the client anyway.

Reply:
That's what I figured, I did just copy the existing code. Cleaned it up a bit in the following commit.